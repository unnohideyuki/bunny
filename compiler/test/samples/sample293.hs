main = do print $ head [1, 2, 3]
          print $ tail [1, 2, 3]
          print $ foldr1 (+) [1, 2, 3, 4]
          print $ take 4 (repeat 'x')
          print $ replicate 3 5
