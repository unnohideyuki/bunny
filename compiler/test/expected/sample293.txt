1
[2,3]
10
"xxxx"
[5,5,5]
